TLM/CSUtil.Commons/LogicUtil.cs

```diff
@@ -1,17 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿namespace CSUtil.Commons {
+    public static class LogicUtil {
+        public static bool CheckFlags(uint flags, uint flagMask, uint? expectedResult = null) {
+            uint res = flags & flagMask;
+            if (expectedResult == null) {
+                return res != 0;
+            }
 
-namespace CSUtil.Commons {
-	public static class LogicUtil {
-		public static bool CheckFlags(uint flags, uint flagMask, uint? expectedResult=null) {
-			uint res = flags & flagMask;
-			if (expectedResult == null) {
-				return res != 0;
-			} else {
-				return res == expectedResult;
-			}
-		}
-	}
+            return res == expectedResult;
```

Comment:
Could this use a ternary operator? Something like...

```csharp
return expected result == null ? res != 0 : expectedResult;
```

Note: I have no clue as to whether there are any performance impacts or other side-effects to using that approach.

Reply:
or
```csharp
return expectedResult == null && res != 0 || expectedResult;
```

Reply:
It could, i've done something like this in SegmentEndId in the previous PR.

Reply:
Done